static/app/views/replays/detail/breadcrumbs/useBreadcrumbFilters.tsx

```diff
@@ -70,6 +70,7 @@ const OPORCATEGORY_TO_TYPE: Record<string, keyof typeof TYPE_TO_LABEL> = {
   'ui.click': 'click',
   'ui.keyDown': 'keydown',
   'ui.input': 'input',
+  feedback: 'feedback',
```

Comment:
oh, i'm noticing there is a `'sentry.feedback': 'feedback',` line just above in this same list. what is that?

Reply:
looks like we added this in here: https://github.com/getsentry/sentry/pull/63883

we originally had this blue breadcrumb showing submitted feedbacks, which i think was coming from the SDK, but i don't see it on new replays anymore. cc @billyvg did these get dropped or do you know why we're not seeing these old feedback breadcrumbs anymore?

Reply:
@michellewzhang They should stilll be there (but only the widget feedbacks, not crash reports) https://github.com/getsentry/sentry-javascript/blob/4ba3e405b01c1f37a64bb3c78b96be0a6cfa2571/packages/replay/src/coreHandlers/util/addFeedbackBreadcrumb.ts#L20-L34

Reply:
weird, i'm not seeing them for widget feedback replays at all

Reply:
resolved for now with https://github.com/getsentry/sentry/pull/67527: we removed the old breadcrumb